recipes/qt/6.x.x/conanfile.py

```diff
@@ -277,27 +277,27 @@ def validate(self):
             raise ConanInvalidConfiguration("alsa and pulseaudio are not supported (QTBUG-95116), please disable them.")
 
     def requirements(self):
-        self.requires("zlib/1.2.11")
+        self.requires("zlib/1.2.12")
         if self.options.openssl:
-            self.requires("openssl/1.1.1m")
+            self.requires("openssl/1.1.1n")
         if self.options.with_pcre2:
-            self.requires("pcre2/10.37")
+            self.requires("pcre2/10.39")
```

Comment:
```suggestion
            self.requires("pcre2/10.37") # needs to be < 10.38
```
Woopsie, we already went through this one earlier in this PR 😊

Reply:
Due to https://github.com/PhilipHazel/pcre2/commit/2410fbe3869cab403f02b94caa9ab37ee9f5854b

I've left a comment few weeks ago to warn against this breaking change in static lib name for Visual Studio: https://github.com/PhilipHazel/pcre2/commit/2410fbe3869cab403f02b94caa9ab37ee9f5854b#commitcomment-68176307